bubblewrap.c

```diff
@@ -314,14 +316,47 @@ propagate_exit_status (int status)
   return 255;
 }
 
+static void
+dump_info (int fd, const char *output, bool exit_on_error)
+{
+  size_t len = strlen (output);
+  if (write (fd, output, len) != len)
```

Comment:
Use `write_to_fd()` here, to handle `EINTR` as well as possible short writes if the info fd is a pipe, etc.

Reply:
Done.